### DISCUSSION TITLE
Auto Generated USDZ flipped Normals

### DISCUSSION THREAD

**Discussion by CrystalDesignMT** [google/model-viewer#4573] [Q&A]
### Description

When opening QuickLook with certain GLB's (exported by BABYLONJS) the auto-generated USDZ has flipped normals, causing Backface-Culling to render wrong faces.
If i flip the Normals on the Model myself it is wrong in the 3D Scene but right in QuickLook AR.

As said this only happens on certain Models but i can't tell why, it certainly is a bug on one side ModelViewer or QuickLook but if you could tell why these Models are affected that could also be sufficient. 

#### Live Demo
https://glitch.com/edit/#!/boatneck-twisty-bellusaurus?path=index.html%3A40%3A0

### Version
- model-viewer: v3.2.1

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by elalish** [comment]
Actually, I would say it's most likely a bug in the three.js USDZExporter, @mrdoob. Thanks for including the repro model!

**Comment by hybridherbst** [comment]
Adding myself here 👀 
There's no bug opened on the three.js side yet, right?

**Comment by elalish** [comment]
Not that I'm aware of.

**Comment by CrystalDesignMT** [comment]
Ok seems like a WebGLRenderer <-> QuickLook thing as explained in the three.js issue where they handle negative scaling differently. 
I guess nothing one can do here other than avoiding negative scaling.
